.github/version-lint.sh

```diff
@@ -27,6 +27,7 @@ done <<< "$FILES_TO_CHANGE"
 MISSING_FILES=$(echo $MISSING_VERSION_FILES | tr '\n' ' ')
 # Check if the variable is not empty
 if [ -n "$MISSING_VERSION_FILES" ]; then
-  echo "### :mega: Workflow files ${MISSING_FILES} are missing actions version tag" >> $GITHUB_OUTPUT
-  echo "### :mega: Workflow files ${MISSING_FILES} are missing actions version tag" >> $GITHUB_STEP_SUMMARY
+  EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
```

Comment:
@vgrassia The lint has been failing when it finds actions that are not pinned for a while now, giving this output: `Unable to process file command 'output' successfully.` That is because, the output that is being sent to the `GITHUB_OUTPUT` and `GITHUB_WORKFLOW_SUMMARY` vars is in-between strings. [This](https://github.com/bitwarden/clients/actions/runs/6014037768) failed few minutes ago.

So when I searched for it, I found [this conversation](https://stackoverflow.com/questions/74137120/how-to-fix-or-avoid-error-unable-to-process-file-command-output-successfully) on StackOverflow, and that's where I found the solution I'm testing.

I used ChatGPT to explain the line you inquired about:

`dd:` This is a command used for copying and converting data. It's often used for low-level operations involving data streams.

`if=/dev/urandom:` This specifies the input file for dd. In this case, it's set to read from the special file /dev/urandom, which provides a stream of random bytes.

`bs=15:` This sets the block size for reading data. In this command, it's set to 15 bytes, meaning dd will read 15 bytes at a time from the input.

`count=1:` This specifies the number of blocks to copy. Since the block size is set to 15 bytes and the count is 1, the command will read 15 bytes in total.

`status=none:` This is used to suppress the status output from dd. It prevents dd from displaying the usual information about the bytes copied.

`base64:` This is a command for encoding binary data into a text-based format that only uses ASCII characters. It's often used to represent binary data as plain text.